<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ARP攻击</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>局域网内主机列表</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>选项</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="1">
           <widget class="QPushButton" name="pBtn_endScan">
            <property name="text">
             <string>结束</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pBtn_startScan">
            <property name="text">
             <string>开始扫描</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>网卡名</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_i">
    <property name="title">
     <string>网卡信息</string>
    </property>
    <addaction name="action_getDev"/>
   </widget>
   <widget class="QMenu" name="menuARP">
    <property name="title">
     <string>ARP攻击</string>
    </property>
    <addaction name="actionARP"/>
   </widget>
   <addaction name="menu_i"/>
   <addaction name="menuARP"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_getDev">
   <property name="text">
    <string>获取网卡信息</string>
   </property>
  </action>
  <action name="actionARP">
   <property name="text">
    <string>ARP攻击</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
